src/App.tsx

```diff
@@ -0,0 +1,7 @@
+import Calculator from 'components/Calculator';
```

Comment:
```suggestion
import { Calculator } from 'components';
```
이런식으로 표현할 수 있도록 index.ts를 활용해주면 좋을 것 같아요!
그러면 import할 때 깔끔하게 관리할 수 있답니다 ㅎㅎ